src/parse.c

```diff
@@ -7501,7 +7505,11 @@ static tree_t p_component_declaration(void)
    pop_scope(nametab);
 
    consume(tEND);
-   consume(tCOMPONENT);
+   if (peek() != tCOMPONENT) {
+      require_std(STD_19, "optional end component");
+      optional(tCOMPONENT);
+   } else
```

Comment:
`else` goes on a new line. Can you try rebasing to see if the clang-format check is fixed now?

Reply:
Rebased and pushed.